WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -1504,6 +1504,19 @@ class ReaderPostDetailFragment : Fragment(),
         return true
     }
 
+    override fun onPageJumpClick(pageJump: String?): Boolean {
+        val jsWasEnabled = readerWebView.settings.javaScriptEnabled
+
+        readerWebView.settings.javaScriptEnabled = true
+
+        readerWebView.evaluateJavascript("document.getElementById('$pageJump').offsetTop") {
+            val yOffset = (readerWebView.scale * it.toFloat()).toInt()
```

Comment:
AS tells me that scale getter is deprecated. Do you think we can use an alternative? StackOverflow suggests `getResources().getDisplayMetrics().density`, but I haven't tried it :)